tox.ini

```diff
@@ -23,3 +23,17 @@ commands =
     flake8 .
     bandit -r .
     mypy pydax
+
```

Comment:
I'm assuming there's a reason we can't add the others in here: `mypy`, `bandit`, and `yaml.lint`. Does tox just not support those config files?